Sources/TuistLoader/Loaders/ManifestLoader.swift

```diff
@@ -177,11 +178,23 @@ public class ManifestLoader: ManifestLoading {
         arguments.append(path.pathString)
         arguments.append("--tuist-dump")
 
-        let result = try System.shared.capture(arguments).spm_chuzzle()
-        guard let jsonString = result, let data = jsonString.data(using: .utf8) else {
-            throw ManifestLoaderError.unexpectedOutput(path)
+        let result = try System.shared.observable(arguments)
+            .do(onNext: { event in
+                switch event {
+                case let .standardError(data):
+                    if let string = String(data: data, encoding: .utf8) {
+                        logger.error("\(string)")
+                    }
+                default:
+                    break
+                }
+            })
+            .toBlocking()
+            .first()
```

Comment:
Something is a bit odd with the observable behavior the output seems to truncate at different lengths - it may require running to completion as opposed to only the first result. 

Re-running a few times:

e.g.

```sh
$ swift run tuist generate
/path/to/tuist/fixtures/ios_app_with_tests/Project.swift
Unexpected output trying to parse the manifest at path /path/to/tuist/fixtures/ios_app_with_tests/Project.swift
```

Other times:

```sh
$ swift run tuist generate
/path/to/tuist/fixtures/ios_app_with_tests/Project.swift:7:47: error: 
Unexpected output trying to parse the manifest at path /path/to/tuist/fixtures/ios_app_with_tests/Project.swift
```

I tried using `.toArray().first` and that helped a bit - but the output was still changing each time I re-ran generate. New lines appear in different locations making the output less helpful - for example when omitting a comma the raw error output usually contains a `^` character that is usually aligned with the location of the missing comma..

Looking at the 1.2.0 code - perhaps we can bring back `error:` to `SystemError.terminated`

```swift
// Tuist 1.2.0
func throwIfErrored() throws {
  // ...
  throw TuistSupport.SystemError.terminated(code: code, error: try utf8stderrOutput())
}
```

Reply:
Figured out the issue. Would you mind trying again @kwridan ?